src/System.Linq.Expressions/src/System/Linq/Expressions/MemberExpression.cs

```diff
@@ -179,6 +179,7 @@ public static MemberExpression Field(Expression expression, FieldInfo field)
         public static MemberExpression Field(Expression expression, string fieldName)
         {
             RequiresCanRead(expression, nameof(expression));
+            ContractUtils.RequiresNotNull(fieldName, nameof(fieldName));
```

Comment:
Is this changing an exception type / message / parameter name / etc. that's getting thrown?